starlette/staticfiles.py

```diff
@@ -108,7 +108,9 @@ def get_path(self, scope: Scope) -> str:
         Given the ASGI scope, return the `path` string to serve up,
         with OS specific path separators, and any '..', '.' components removed.
         """
-        return os.path.normpath(os.path.join(*scope["path"].split("/")))
+        return typing.cast(
+            str, os.path.normpath(os.path.join(*scope["path"].split("/")))
+        )
```

Comment:
I think this is missing from https://github.com/python/typeshed/blob/main/stdlib/os/__init__.pyi isn't it?

Can we instead add an `type: ignore` here, and open an issue there or something?

Reply:
As `scope["path"]` is `Any`, the signature of `os.path.join` and `os.path.normpath` doesn't match anymore, thus the return type was `Any`. Switched to a `type: ignore` instead.